integration-cli/docker_cli_service_create_test.go

```diff
@@ -103,3 +120,53 @@ func (s *DockerSwarmSuite) TestServiceCreateWithSecretSourceTarget(c *check.C) {
 	c.Assert(refs[0].File, checker.Not(checker.IsNil))
 	c.Assert(refs[0].File.Name, checker.Equals, testTarget)
 }
+
+func (s *DockerSwarmSuite) TestServiceCreateMountTmpfs(c *check.C) {
+	d := s.AddDaemon(c, true, true)
+	out, err := d.Cmd("service", "create", "--mount", "type=tmpfs,target=/foo", "busybox", "sh", "-c", "mount | grep foo && sleep 60")
```

Comment:
nit: `tail -f /dev/null` would be better than `sleep 60`
Also, I think we should remove this service on exit